### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by jonathastavares**
In this milestone we added:

- Tests for calculate.js
- Tests for operate.js

Built with:
- Jest
- React Testing Library

### CODE REVIEW

File: src/__Tests__/calculate.test.js [hunk 1]
```diff
@@ -0,0 +1,211 @@
+import calculate from '../logic/calculate';
+
```
Comment (rloterh) [inline src/__Tests__/calculate.test.js:2]: - [ ] One test suite is failing. Kindly ensure to add some test cases in the `App.test.js` file, or remove where appropriate.

**Failing test suite:**
image

Comment (rloterh) [inline src/__Tests__/calculate.test.js:2]: - [ ] Jest snapshots are missing in your test. Kindly ensure that Jest snapshots are used for testing **all** the components.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Marcos & Jonathas, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## Thank you for implementing all the project requirements, you've done a great job :trophy: There is nothing else to say other than... it's time to merge it :champagne:  :tada: Congratulations

Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**